go-tour/stringer_test.go

```diff
@@ -0,0 +1,21 @@
+package stringer
+
+import (
+	"testing"
+)
+
+func BenchmarkStrConvItoa(b *testing.B) {
+	for i := 0; i < b.N; i++ {
+		for _, ip := range hosts {
```

Comment:
this will cause additional costs for iteration, benchmark don't care about the result - it more about performance